share/spice/hex/hex.js

```diff
@@ -84,4 +80,8 @@
             });
         });
     };
-}(this));
\ No newline at end of file
+
+    Handlebars.registerHelper('hex_plural', function(number, options) {
```

Comment:
@moollaza Actually, after reading about the [Handlebars helpers](http://docs.duckduckhack.com/frontend-reference/handlebars-helpers.html) that you offer, the official `plural` helper was my first option, but that helper returns the context or number plus the pluralized word, and in this case, IMHO that looks weird. Consider the following examples:

* You may also be interested in the following **1 link**:
* You may also be interested in the following **3 links**:

That's why I create the custom `hex_plural` helper. Wdyt?

BTW, thanks for the source code reference :+1: